test/docker-executeContainerCommand.test.js

```diff
@@ -0,0 +1,13 @@
+const test = require('ava');
+const {createDefaultContainer} = require('./helpers/createDefaultContainer');
+const {executeContainerCommand} = require('../src/docker');
+
+test('response includes stdout, stderr, and inspectOutput with ExitCode 0', async (t) => {
+  const container = await createDefaultContainer();
+  await container.start();
```

Comment:
Should you stop the container in a `finally` block to avoid leaking containers?

Reply:
oops, thanks, added the finally block